features/comments_and_kudos/guest_comments.feature

```diff
@@ -43,3 +43,26 @@ Scenario: Guest comments with embedded images are rendered as plain text
     And I should see "alt="
     And I should see "baz"
     But I should not see the image "src" text "https://example.com/image.jpg"
+
+Scenario: Guest sees warning footnote and required fields on comment form
+  Given the work "Test Work" by "author" with guest comments enabled
+  When I go to the work "Test Work"
+  Then I should see "You will not be able to edit or delete your comment after it is posted."
+  And I should see "Guest name (required)"
+  And I should see "Guest email (required)"
+  And I should see "All fields are required. Your name and comment text will both be publicly displayed. Your email address will not be made public, but it will be used to send you notifications of any replies to your comment."
```

Comment:
While you're in this file for the other review comment, please indent the "And" steps one extra level (also in the other scenario)